### PR TITLE
Update Docs to New Verificaton Process

### PR DESCRIPTION
**Pull request by kd9lsv**

### CODE REVIEW

File: content/docs/ve/getExamtoolsAccount.md [hunk 1]
```diff
@@ -19,9 +19,13 @@ The [ExamTools ecosystem]({{<relref "../general/websiteDecoder.md">}}) has two d
...
-* Login with your newly created account and note the RequestID in the middle of the page.
-![Exam.Tools Request ID](../images/examtoolsRequestID.png)
-* Fill out [this form](https://forms.gle/cD9fTqv5ognik6XC9).
+* Login with your newly created account and note the instructions on how to verify your ExamTools account in the middle of the page.
+* Once logged in, head to your profile to verify your ExamTools account.
+![ExamTools Getting Started](../images/examtoolsGettingStarted.png)
+* Carefully fill out the Verify Callsign form.
+![ExamTools Verify Account Instructions](../images/examtoolsVerifyCallsign.png)
+* Wait for an email from Examtools stating your account has been verified.
+![ExamTools Account Apporved](../images/examtoolsEmailApporval.png)
```
Comment (taxilian) [inline content/docs/ve/getExamtoolsAccount.md:28]: "Approved/Approval", not "Apporved/Apporval" -- rename file and fix text =]

File: content/docs/ve/getSandboxAccount.md [hunk 2]
```diff
@@ -19,7 +19,12 @@ The [ExamTools ecosystem]({{<relref "../general/websiteDecoder.md">}}) has two d
 * Enter your personal details.  For clarity, we recommend using your call sign as the username.
 ![ExamTools Create Profile](../images/examtoolsCreateAccount.png)
 * It will then redirect you to [ExamTools](https://examtools.dev/portal/veLogin).
-* Fill out [this form](https://forms.gle/NiAeLFpX7R7E7NZC9).
+* Once logged in, head to your profile to verify your ExamTools account.
+![ExamTools Getting Started](../images/examtoolsGettingStarted.png)
+* Carefully fill out the Verify Callsign form.
+![ExamTools Verify Account Instructions](../images/examtoolsVerifyCallsign.png)
+* Wait for an email from Examtools stating your account has been verified.
+![ExamTools Account Apporved](../images/examtoolsEmailApporval.png)
```
Comment (taxilian) [inline content/docs/ve/getSandboxAccount.md:27]: "Approved/Approval", not "Apporved/Apporval" -- rename file and fix text =]

**Review by taxilian — COMMENTED** [review]
Looks good, just needs a couple of typos fixed

File: content/docs/ve/getSandboxAccount.md (hunk 2 above)
Comment (kd9lsv) [reply content/docs/ve/getSandboxAccount.md]: Fixed by commit 14dfe31

File: content/docs/ve/getExamtoolsAccount.md (hunk 1 above)
Comment (kd9lsv) [reply content/docs/ve/getExamtoolsAccount.md]: Fixed by commit 14dfe31
@taxilian

**Comment by taxilian** [comment]
thanks Connor!

**Pull request merged**